docs/developer/stock/overview.mdx

```diff
@@ -0,0 +1,321 @@
+This mutation is used to update the existing stock levels of a specific product variant in one or more warehouses. 
+You can modify the quantity of a variant at particular locations using variant ID or sku.
+
+- [`productVariantStocksDelete`](/api-reference/products/mutations/product-variant-stocks-delete.mdx):
+This mutation is used to remove all stock information for a specific product variant from a particular warehouse. 
+This effectively indicates that the variant is no longer stocked at that location.
+
+### Bulk Stock Management
+
+For managing stock levels across multiple product variants and warehouses efficiently, 
+Saleor offers bulk operations. Refer to the [Update Stocks in Bulk](/developer/bulks/bulk-stock.mdx) documentation for more details.
+
+
+### Querying Stocks
+Saleor provides stock querying capabilities that help you:
+- Monitor inventory levels across warehouses
+- Track stock allocations and reservations
+- Manage product availability
+- Implement inventory alerts and notifications
+
+#### Get All Stocks
+You can use the [`stocks`](/api-reference/products/queries/stocks.mdx) query to get all stocks or apply a search filter to get only stocks for variants or a warehouse with a specific name.
+
+```graphql 
+query Stocks{
+  stocks(first:10, filter: {search:"Europe"}){
+    totalCount
+    edges{
+      node{
+        id
+        quantity
+        warehouse{
+          id
+          name
+          slug
+        }
+        productVariant{
+          id
+          name
+        }
+      }
+    }
+  }
+}
+```
+
+#### Get Single Stock
+To retrieve detailed information about a specific [`stock`](/api-reference/products/queries/stock.mdx) entry:
+
+```graphql
+query Stock {
+  stock(id: "") {
+    id
+    quantity
+    warehouse {
+      id
+      name
+    }
+    productVariant {
+      id
+      name
+    }
+  }
+}
+```
+
+#### Get Warehouse Stocks
+To view all stock entries for a specific [`warehouse`](/api-reference/products/queries/warehouse.mdx), including detailed product information and quantity:
+
+```graphql
+query WarehouseStocks{
+  warehouse(id: "V2FyZWhvdXNlOjI1YzdlZDNlLTljYjQtNDUzNi04ZGNjLTgzZTdlNDIzZTNjMQ=="){
+    stocks(first:10){
+      totalCount
+      edges{
+        node{
+          id
+          warehouse{
+            id
+            name
+          }
+          productVariant{
+            id
+            sku
+            name
+            product{
+              id
+              name
+            }
+          }
+          quantity
+          quantityAllocated
+          quantityReserved
+        }
+      }
+    }
+  }
+}
+```
+
+## Webhooks
+
+Saleor provides several webhooks to help you monitor and react to stock-related events:
+
+- **PRODUCT_VARIANT_OUT_OF_STOCK**: Triggered when a product variant's stock level reaches zero, allowing you to implement low-stock notifications or automatic reordering.
```

Comment:
```suggestion
- `PRODUCT_VARIANT_OUT_OF_STOCK`: Triggered when a product variant's stock level reaches zero, allowing you to implement low-stock notifications or automatic reordering.
```

The same goes to all the other webhook names. Since these are "code-related", I would use the code block.